bundle/src/main/java/com/cognifide/cq/cqsm/api/history/Entry.java

```diff
@@ -19,31 +19,30 @@
  */
 package com.cognifide.cq.cqsm.api.history;
 
-import com.google.common.collect.ComparisonChain;
 
 import com.cognifide.cq.cqsm.api.executors.Mode;
 import com.cognifide.cq.cqsm.api.logger.ProgressEntry;
+import com.cognifide.cq.cqsm.core.history.HistoryResourceAdapter;

```

Comment:
API should not user classes from core. Is this fixed in other pull request  ? If not, I would say that we need to change it now.

Reply:
Ok, I moved core-related classes to API: that was more suitable place for them, thanks.